src/test/java/org/broadinstitute/hellbender/tools/spark/sv/FindBreakpointEvidenceSparkUnitTest.java

```diff
@@ -55,7 +55,7 @@ public void getQNamesTest() {
         final Set<String> actualQNames = new HashSet<>();
         FindBreakpointEvidenceSpark.getQNames(params, ctx, broadcastMetadata, expectedIntervalList, mappedReads)
                 .stream()
-                .map(FindBreakpointEvidenceSpark.QNameAndInterval::getKey)
+                .map(QNameAndInterval::getKey)
```

Comment:
Set comparison is implemented in a rather silly way in the java library.  We need to make sure we have an O(1) get function.  Collectors.toSet makes no guarantees what kind of set you'll get.  Assert.assertEquals makes no guarantee about which set will be used for iteration and which will be used for lookup.  Only safe thing to do is to pin down the types.
This is a trifling worry for this little unit test, but it's just my habit.